update TiDB 6MB limit (#5676)
pingcap/docs: error-codes.md

* Error Number: 8025

>     The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB by default is 6 MB.

Reviewer: ```suggestion
    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB is 6 MB by default.
```

>     If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item to relax the limit.

Reviewer: ```suggestion
    If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration value to relax the limit.
```

>     If a pair exceeds this limit, you need to properly adjust the configuration items in [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) to ease the restrictions.

Reviewer: `txn-entry-size-limit` is an configuration item.
```suggestion
    If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item to relax the limit.
```
